tests/components/test-class-component.php

```diff
@@ -417,17 +417,19 @@ public function test_children_from_config() {
 			'example/parent',
 			[
 				'children' => [
-					[ 'example/child-1' ],
-					'example/child-2',
+					'Text node.',
```

Comment:
This test asserts that we are now supporting a couple different syntaxes when adding children.

1. Text nodes remain text.
2. An array with a single value can represent a component type.
3. An array with a `name` key will turn into a component of that type.